sonaranalyzer-dotnet/its/expected/Nancy/Nancy-{34576216-0DCA-4B0F-A0DC-9075E75A676F}-S3900.json

```diff
@@ -1848,32 +1848,6 @@
 },
 {
 "id":  "S3900",
-"message":  "Refactor this method to add validation of parameter 'modelType' before using it.",
```

Comment:
Since all SE checks run simultaneously we could generate much bigger exploded graphs, thus creating some FNs in large methods. We could investigate the further increase of the EG limits in the future.